main.js

```diff
@@ -49,41 +50,39 @@ var textInput = document.querySelector('.message-input');
 var textInputLabel = document.querySelector('.input-error');
 var errorMessage = document.querySelector('.receive-error');
 var clearMessageButton = document.querySelector('.clear-message');
-var currentMessage = '';
 
 //--------------------/event listeners/----------------------//
-// window.addEventListener('load', )
+
 receiveMessageButton.addEventListener('click', displayMessage);
 addMessageButton.addEventListener('click', displayForm);
 submitButton.addEventListener('click', submitMessage);
+clearMessageButton.addEventListener('click', showBellImage);
 addMantraSelect.addEventListener('click', hideErrorMessage);
 addAffirmationSelect.addEventListener('click', hideErrorMessage);
 mantraSelect.addEventListener('click', hideErrorMessage);
 affirmationSelect.addEventListener('click', hideErrorMessage);
-clearMessageButton.addEventListener('click', showBellImage);
 
 //--------------------/functions/----------------------//
-//transition to the data model!
-//create helper function that checks to see if newAff/newMant string exists in the aff or mant arrays.
-//create logic statement to pass it to the correct array accordingly.
 
 function submitMessage(e) {
   e.preventDefault();
   hideForm();
-  if (!addMantraSelect.checked && !addAffirmationSelect.checked) {
+  if (!addMantraSelect.checked && !addAffirmationSelect.checked || !textInput.value) {
     displayForm();
     textInputLabel.classList.toggle('hidden');
 
   } else if (addAffirmationSelect.checked) {
     textInputLabel.classList.add('hidden');
```

Comment:
textInputLabel.classList.add('hidden') is a bit hard to read/understand. Think about creating two helper functions to show and hide elements. These helper functions should have a parameter of whatever element you want to show or hide.

Reply:
Great idea! This makes it much easier to read what my main functions are doing.